### DISCUSSION TITLE
Independents friendly_name and entity_id

### DISCUSSION THREAD

**Discussion by outofsight** [zachowj/node-red-contrib-home-assistant-websocket#644] [Feature Request]
Hello,

it would be useful to be able to set `fliendly_name` and `entity_id` in an independent way. 
This apply to sensor, switch, ...

Currently we can set `friendly_name` and `entity_id` will be the slugified version of it.

It would be useful to keep `entity_id` to some internal english programmatic identifier but the `friendly_name` to a end-user localized string.

We can do this with `customize:` in Home Assistant but it's more complicate.

**Comment by zachowj** [comment]
the `entity_id` is only set on the first the deploy. Changing the field after the first deployment will only change the `friendly_name`.

**Comment by outofsight** [comment]
Interesting, thanks, I'll use this way in the meanwhile.

But I think it would be useful to have two "fields" in the node, one to document the correct `entity_id` to reference in HA/YAML, and another to view/set/change the default `friendly_name`.
